### PR TITLE
feat: add raise hand to members

### PR DESCRIPTION
**Pull request by cscheffauer**
# COMPLETES 
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-337885

## This pull request addresses

- Adding the raiseLowerHand action to the members part in plugin-meetings
- Added tests

## by making the following changes

Similar to mute action

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

automated unit tests

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/members/index.js [hunk 1]
```diff
@@ -739,6 +739,26 @@ export default class Members extends StatelessWebexPlugin {
...
 
+  /**
+   * Raise or lower the hand of a member in a meeting
+   * @param {String} memberId
+   * @param {boolean} [raise] - to raise hand (=true) or lower (=false), default: true
+   * @returns {Promise}
+   * @public
+   * @memberof Members
+   */
+  raiseLowerHand(memberId, raise = true) {
```
Comment (maxinteger) [inline packages/node_modules/@webex/plugin-meetings/src/members/index.js:750]: minor: I think the name of the function is not descriptive enough
`raiseOrLowerHand`, `toggleRaiseHand` or `setRaiseHandState`

File: packages/node_modules/@webex/plugin-meetings/src/members/request.js [hunk 2]
```diff
@@ -63,6 +63,16 @@ export default class MembersRequest extends StatelessWebexPlugin {
 
     return this.request(requestParams);
   }
+  
+  raiseLowerHandMember(options) {
```
Comment (maxinteger) [inline packages/node_modules/@webex/plugin-meetings/src/members/request.js:67]: minor: I think the name of the function is not descriptive enough
`raiseOrLowerHandMember` or `setRaiseHandMemberState`

**Review by maxinteger — CHANGES_REQUESTED** [review]
beside some minor comments looks good

**Review by Coread — CHANGES_REQUESTED** [review]
Can you add to the documentation in packages/node_modules/@webex/plugin-meetings/README.md

File: packages/node_modules/@webex/plugin-meetings/src/members/index.js (hunk 1 above)
Comment (cscheffauer) [reply packages/node_modules/@webex/plugin-meetings/src/members/index.js]: adressed - pls check again

File: packages/node_modules/@webex/plugin-meetings/src/members/request.js (hunk 2 above)
Comment (cscheffauer) [reply packages/node_modules/@webex/plugin-meetings/src/members/request.js]: adressed - pls check again

**Review by Coread — APPROVED** [review]

**Review by maxinteger — APPROVED** [review]
Nice job

**Pull request merged**